test/modules/core/shaderlib/shadow/shadow.spec.js

```diff
@@ -5,52 +5,50 @@ import {shadow} from '@deck.gl/core/shaderlib';
 import {Matrix4, Vector3} from 'math.gl';
 import {PROJECT_COORDINATE_SYSTEM} from '@deck.gl/core/shaderlib/project/constants';
 
+// unproject viewport corners to ground and use them as reference points to test tight fitting bounding box
```

Comment:
Define test viewports with the test cases so that future maintainers know where they come from.